test/elixir_conf_africa/events_test.exs

```diff
@@ -22,11 +24,75 @@ defmodule ElixirConfAfrica.EventsTest do
       assert Events.list_events() == [event]
     end
 
+    test "get_elixir_conf_event /0 returns the elixir conf event" do
+      valid_attrs = %{
+        name: "ElixirConf Africa 2024",
+        description: "some description",
+        location: "some location",
+        event_type: "some event_type",
+        start_date: ~N[2023-10-05 06:18:00],
+        end_date: ~N[2023-10-05 06:18:00]
+      }
+
+      assert {:ok, %Event{} = event} = Events.create_event(valid_attrs)
+      assert Events.get_elixir_conf_event() == event
+    end
+
     test "get_event!/1 returns the event with given id" do
       event = event_fixture()
       assert Events.get_event!(event.id) == event
     end
 
+    test "get_elixir_conf_event_and_ticket_types/0 returns the elixir conf event with ticket types" do
+      valid_attrs = %{
+        name: "ElixirConf Africa 2024",
+        description: "some description",
+        location: "some location",
+        event_type: "some event_type",
+        start_date: ~N[2023-10-05 06:18:00],
+        end_date: ~N[2023-10-05 06:18:00]
+      }
+
+      assert {:ok, %Event{} = event} = Events.create_event(valid_attrs)
```

Comment:
There is alot of redundancy here maybe we should introduce factories.

WDYT?

Reply:
I have not interacted with them before , is it this https://hexdocs.pm/ecto/test-factories.html ?

Reply:
Precisely

Reply:
Hey @okothkongo   , Thanks for this , I read it over the weekend and got to understand its benefits , I have implimented it in the changes below , Thanks